MaterialComponents.podspec

```diff
@@ -956,6 +956,26 @@ Pod::Spec.new do |mdc|
     end
   end
 
+  mdc.subspec "Elevation" do |component|
+    component.ios.deployment_target = '9.0'
+    component.public_header_files = "components/#{component.base_name}/src/*.h"
+    component.source_files = [
+    "components/#{component.base_name}/src/*.{h,m}",
+    "components/#{component.base_name}/src/private/*.{h,m}"
+    ]
+    component.resources = [
+    "components/#{component.base_name}/src/Material#{component.base_name}.bundle"
+    ]
+
+    component.test_spec 'UnitTests' do |unit_tests|
+      unit_tests.source_files = [
+      "components/#{component.base_name}/tests/unit/*.{h,m,swift}",
+      "components/#{component.base_name}/tests/unit/supplemental/*.{h,m,swift}"
+      ]
+      unit_tests.resources = "components/#{component.base_name}/tests/unit/resources/*"
```

Comment:
i don't believe there are any resources, can this be removed?

Reply:
Removed.